springwolf-core/src/test/java/io/github/springwolf/core/integrationtests/AsyncApiDocumentIntegrationTest.java

```diff
@@ -54,7 +54,7 @@ void asyncListenerAnnotationIsFound() {
                             "listener-channel_receive_listen2",
                             "listener-channel_receive_listen3",
                             "listener-channel_receive_listen4",
-                            "listener-class-channel_receive_ClassListener");
+                            "listener-class-channel_receive_listen");
```

Comment:
Class level annotations use the first method as an identifier (simpler code compared to passing down the class name)